nautobot/docs/development/navigation-menu.md

```diff
@@ -0,0 +1,129 @@
+# Navigation Menu Tabs
```

Comment:
Need to add a reference to this new file in `mkdocs.yml` so it shows up in the table of contents.